README.md

```diff
@@ -161,6 +301,243 @@ more metric, you must run CASTanet for the first metric
 and then run CASTanet subsequently for
 each additional metric (with the corresponding CLI command).
 
+### CASTanet as a Python Library
+
+CASTanet is also available on PyPI to be used as a Python
+library. Find it [here](https://pypi.org/project/castanet/).
+With the CASTanet library, a user is able to investigate
+their Python files with many different function calls.
+Specifically, CASTanet is broken down into two parts:
+
+1. `generate_trees`: Traverses a directory and generates
+concrete-abstract-syntax trees of Python files using
+LibCST
+1. `counter`: Uses concrete-abstract-syntax-trees to
+calculate metrics associated with the contents of a Python
+module
+
+### counter
+
+`from castanet import counter`
+
+#### sum_dict_vals
+
+Calculate the sums of values from dictionaries.
+Called to get number values from the result of a function.
+
+`counter.sum_dict_vals(cast_dict)`
+
+_ARGUMENTS:_
+
+* cast_dict: A dictionary of files and corresponding CASTs
```

Comment:
Outdated argument description. Might make sense to rename the parameter since it doesn't contain CSTs.